corehq/apps/linked_domain/views.py

```diff
@@ -238,12 +241,15 @@ def page_context(self):
         master_reports, linked_reports = get_reports(self.domain)
         master_keywords, linked_keywords = get_keywords(self.domain)
 
+        is_superuser = self.request.couch_user.is_superuser
```

Comment:
Is this the best way to obtain whether user is a superuser?

Reply:
yep